crates/runtime/src/flight.rs

```diff
@@ -241,6 +253,19 @@ impl Service {
 
         Ok((flights_stream.boxed(), query_result.from_cache))
     }
+
+    fn get_user_agent<T>(req: &Request<T>) -> [KeyValue; 3] {
```

Comment:
```suggestion
    fn get_request_labels<T>(req: &Request<T>) -> [KeyValue; 3] {
```

Reply:
If we wanted to extend the labels we extract from a request, it should go in this function - so naming it appropriately up front seems wise.